Makefile

```diff
@@ -2,6 +2,7 @@
 SHELL := /bin/bash -o pipefail -o errexit -o nounset
 
 addlicense=go run github.com/google/addlicense@v1.0.0
+buildflags=-ldflags "-X github.com/mineiros-io/terramate.Version=$(VERSION)"
```

Comment:
VERSION not defined.

Reply:
In this case it sets the Version to "", which then uses the default (same behavior as not passing any build flag, which needs to work too).

Reply:
It runs this:

https://github.com/mineiros-io/terramate/runs/4486806891?check_suite_focus=true#step:5:6

which produces:

https://github.com/mineiros-io/terramate/runs/4486806891?check_suite_focus=true#step:6:6

which is the default version when a version is not defined.

Reply:
which is wrong as this is version 0.0.3 somethng

Reply:
all version constraint checks will fail...